File: app/src/main/java/com/rkbapps/neetflix/models/person/movie/WorkForMovies.java
package com.rkbapps.neetflix.models.person.movie;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class WorkForMovies {

    @SerializedName("cast")
    @Expose
    private List<AsACastModel> cast;
    @SerializedName("crew")
    @Expose
    private List<AsACrewModel> crew;
    @SerializedName("id")
    @Expose
    private Integer id;

    public List<AsACastModel> getCast() {
        return cast;
    }

    public List<AsACrewModel> getCrew() {
        return crew;
    }

    public Integer getId() {
        return id;
    }
}

File: app/src/main/java/com/rkbapps/neetflix/models/person/movie/WorkForMovies.kt
package com.rkbapps.neetflix.models.person.movie

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class WorkForMovies(
    @SerializedName("cast")
    @Expose
    val cast: List<AsACastModel>,
    @SerializedName("crew")
    @Expose
    val crew: List<AsACrewModel>,
    @SerializedName("id")
    @Expose
    val id: Int
)